build/build.net.tourbook/build.gradle

```diff
@@ -159,33 +152,26 @@ task mt_40_ExtractWinFiles {
 /*
  Create win exe files
 */
-task mt_50_CreateExeFiles {
-        
-   doLast {
-       
-      println 'Creating Windows setup files'
+task mt_50_CreateExeFiles(type: Exec) {
```

Comment:
Only one windows installer is created instead of two

<img width="2089" height="362" alt="image" src="https://github.com/user-attachments/assets/de3e7e1d-a45b-4d4d-b551-e6ebfc63ddb3" />


The build process runs without any errors

<img width="1310" height="755" alt="image" src="https://github.com/user-attachments/assets/a87da2fb-f201-48c8-96ac-f57b690223c1" />

Reply:
I will try locally, it works on the GitHub build

Reply:
I had to switch machine recently for my MT development and need to install maven etc.... I will let you know once that is done

Reply:
My build fails with the error below. Any idea why ?

<img width="2533" height="1493" alt="image" src="https://github.com/user-attachments/assets/24d39f84-df4f-4498-a92f-d992a1a8ea3c" />

Reply:
There is a local and reduced git repo to speedup the build process. The startingpoint is this readme file https://github.com/mytourbook/mytourbook/blob/main/build/build.eclipse.4.26.nl.repo/How%20to%20update%20this%20p2%20repo.txt

Reply:
The build do not run, it creates these logs

[mt-build.log](https://github.com/user-attachments/files/22229883/mt-build.log)
[log.txt](https://github.com/user-attachments/files/22229886/log.txt)

Reply:
In your screenshot, the `.exe` files are much larger than when I created it. Because the build do not run, so I can't can say how large my `.exe` files will be

Reply:
... and the gradle version

<img width="3001" height="350" alt="image" src="https://github.com/user-attachments/assets/3d540962-9091-4358-aa38-55918cd2ba97" />

Reply:
With tycho 4.0.13 the build runs without an error and the .exe file are not much larger than before

[mt-build.log](https://github.com/user-attachments/files/22237757/mt-build.log)

<img width="1005" height="327" alt="image" src="https://github.com/user-attachments/assets/9ce4c506-5e1b-460a-b972-bc4e10caa7a0" />

Reply:
> ... and the gradle version
> 
Yes, but I had already upgraded it the last time you tried to run the build.

>With tycho 4.0.13 the build runs without an error and the .exe file are not much larger than before

I can downgrade the tycho build if you prefer but it runs fine on my Windows and on the GutHub build

Reply:
>With tycho 4.0.13 the build runs without an error and the .exe file are not much larger than before

Can you try with Tycho 4 to see if Tycho 5 is actually the culprit ? I can't reproduce it locally, it works fine for me

Reply:
> > With tycho 4.0.13 the build runs without an error and the .exe file are not much larger than before
> 
> Can you try with Tycho 4 to see if Tycho 5 is actually the culprit ? I can't reproduce it locally, it works fine for me

This is what I already did.

I found the same error in https://github.com/eclipse-tycho/tycho/issues/4226 with the hint that a newer maven version can fix it, and it fixed this issue. Now the build with tycho 5 is working 😄